config/features.yml

```diff
@@ -377,6 +377,12 @@ features:
     actor_type: user
     description: Enables secure messaging
     enable_in_development: true
+  grandfathering_dsl:
```

Comment:
I'm wondering if we could pick a name that's more descriptive of the functionality being 'enabled/disabled' here. I'm not a fan of the term 'grandfathering' in technical places because it not really a well-defined term.

Reply:
Do you have some suggestions? I wasn't too happy with the name either but I couldn't come up with something clear enough. Maybe something like `bind_dsl_account_to_login_gov_account`? 🤔 I don't really like that one either... Naming is so hard...

Reply:
What exactly will the flag itself be used for? Will it be whether or not to show a frontend modal that starts a 'grandfathering' flow?